### PR TITLE
update controls import

### PR DESCRIPTION
**Pull request by RaulPROP**
I have updated the controls so the users can access new features as in [this issue](https://github.com/vasturiano/3d-force-graph/issues/317).

I also added a .gitignore file.

### CODE REVIEW

File: src/three-render-objects.js [hunk 1]
```diff
@@ -43,11 +43,9 @@ const three = window.THREE
   Clock
 };
 
-import ThreeTrackballControls from 'three-trackballcontrols';
-import OrbitControlsWrapper from 'three-orbit-controls';
-const ThreeOrbitControls = OrbitControlsWrapper(three);
-import FlyControlsWrapper from 'three-fly-controls';
-const ThreeFlyControls = (FlyControlsWrapper(three), three.FlyControls);
+import { TrackballControls as ThreeTrackballControls } from 'three/examples/jsm/controls/TrackballControls.js';
+import { OrbitControls as ThreeOrbitControls } from 'three/examples/jsm/controls/OrbitControls.js';
```
Comment (sirrodgepodge) [inline src/three-render-objects.js:47]: can't you just import from `three`?  not the example code?  The actual `three` dependency is up to date btw (I checked that when trying to debug)

Comment (RaulPROP) [reply src/three-render-objects.js:47]: I tried to, but node couldn't find the object. The only part of the code where the controls are exported are on the example folder.

Also I did a fast research and found that [on the threejs examples](https://github.com/mrdoob/three.js/blob/master/examples/misc_controls_fly.html#L36) they export them from that path.

Comment (sirrodgepodge) [reply src/three-render-objects.js:47]: wow looks like it, opened an issue here: https://github.com/mrdoob/three.js/issues/19250

Comment (sirrodgepodge) [reply src/three-render-objects.js:47]: okay it actually looks like this is the approved way to do it:
https://github.com/mrdoob/three.js/issues/19250#issuecomment-620944980

**Comment by vasturiano** [comment]
Thanks @RaulPROP for this. I think it would be certainly a good shift to consume the controls directly from the `three` package.

However this is a bit of a problematic approach, because `three` is not a dependency of this module, but a **peerDependency**. That means it's not guaranteed to be available for import at bundle time and is expected to be provided by the consumer.

In the UMD version this takes the form of the consumer [including a script tag](https://github.com/vasturiano/three-render-objects/blob/master/example/basic/index.html#L4) which exposes a [global understood by the bundler](https://github.com/vasturiano/three-render-objects/blob/master/rollup.config.js#L14) (rollup).

Since the controls are not exposed as part of the core three build, things gets a bit complicated. We could either:

1) Shift `three` to a regular dependency instead of `peerDependency`. This is highly undesirable as it would over bloat the module bundle size. It would also prevent the consumer from having a single instance of `three` in the application, since it's very likely this lib will be used side by side with other `three` functionality.

2) Expect the UMD consumers to include additional script tags for all the controls used in the module, and register all of those as expected globals in rollup. This requirement would be confusing for the users imo, and would lead to quite a bit of head-scratching and unexpected hard to debug linking issues, because the need for it is not really apparent.

3) Include `three` as `devDependency` in addition to `peerDependency`. This is probably the most promising of all the options, but it is kind of hackish as that's not really the main use of devDependencies. And I'm also not 100% sure is not gonna lead into dependency conflicts upstream.

I'm inclined to give 3) a try, unless you can think of a better way to approach it.

Hopefully it's clear why this is a less straight-forward change than it immediately appears. This is the reason why I initially went the route of importing the controls from separate packages, because you can then separate the peer from the regular dependencies.

**Comment by RaulPROP** [comment]
Oh, I didn't notice this problem.

I don't know which one is the best solution, I don't have much experience dealing with this situations.

Maybe we could create a fork of the libraries that this package is currently using and update the code to match the three.js/examples ones?

**Comment by vasturiano** [comment]
Ok I went with option 3). Also removed the stand-alone dependencies since they're no longer used.

Thanks!

**Pull request merged**